cmd/livepeer_cli/wizard_ticketbroker.go

```diff
@@ -119,12 +90,23 @@ func (w *wizard) unlock() {
 	}
 
 	fmt.Printf("Current Deposit: %v\n", eth.FormatUnits(sender.Deposit, "ETH"))
-	fmt.Printf("Current Penalty Escrow: %v\n", eth.FormatUnits(sender.PenaltyEscrow, "ETH"))
+	fmt.Printf("Current Reserve: %v\n", eth.FormatUnits(sender.Reserve, "ETH"))
+
+	if sender.ReserveState == pm.Frozen {
+		currRound := w.currentRound()
+
+		fmt.Printf("Current Round: %v\n", currRound)
+		fmt.Printf("Thaw Round: %v\n", sender.ThawRound)
+
+		fmt.Printf("Cannot deposit because sender's reserve is frozen and not yet thawed")
```

Comment:
Refresh my memory : we're only allowing one reserve at a time? So they can't deposit a fresh reserve if their old one has become frozen?

Reply:
Yeah this is the case at the moment - if a sender's reserve is frozen the sender will be unable to do anything until the reserve is thawed. I'm still thinking about whether to change this in the contracts, but I think we can revisit this separately